AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_ExposureControlAdded.py

```diff
@@ -149,35 +239,285 @@ def AtomEditorComponents_ExposureControl_AddedToEntity():
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('Enable'), False)
+        Report.result(
+            Tests.toggle_enable_parameter_off,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('Enable')) is False)
+        # Set the Enable parameter to True.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('Enable'), True)
+        Report.result(
+            Tests.toggle_enable_parameter_on,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('Enable')) is True)
+
+        # 9. Toggle the Enable Override parameter.
+        # Set the Enable Override parameter to False.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('Enabled Override'), False)
+        Report.result(
+            Tests.toggle_enabled_override_off,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('Enabled Override')) is False)
+        # Set the Enable Override parameter to True.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('Enabled Override'), True)
+        Report.result(
+            Tests.toggle_enabled_override_on,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('Enabled Override')) is True)
+
+        # 10. Toggle the ExposureControlType Override parameter.
+        # Set the ExposureControlType Override parameter to False.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('ExposureControlType Override'), False)
+        Report.result(
+            Tests.toggle_control_type_override_off,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('ExposureControlType Override')) is False)
+        # Set the ExposureControlType Override parameter to True.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('ExposureControlType Override'), True)
+        Report.result(
+            Tests.toggle_control_type_override_on,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('ExposureControlType Override')) is True)
+
+        # 11. Set ManualCompensation Override to min/max values.
+        # Set ManualCompensationOverride to minimum value.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('ManualCompensation Override'), value=0.0)
```

Comment:
nit: how come some `set_component_property_value()` calls use the `value=` kwarg but others don't? we should be consistent throughout the file

Reply:
I was basing this on the method that Scott used in his Display Mapper PR - Boolean values don't actually show "value=" before setting True or False.

Reply:
it can have it or not have it, but the whole file should be consistent - either all use the `value` kwarg that are calling `set_component_property_value()` or none of them use the `value` kwarg

its just a consistency/styling that anyway, not a big deal - boolean can be set with `value=True` if using `set_component_property_value()` also - `value` is a kwarg (keyword arg) for that function

Reply:
I'm opting to go with value= for all value assignments.